### PR TITLE
"Clarify the 'Existing Projects' section of the TypeScript docs:

### PR DESCRIPTION
**Pull request by ShaunFerris**
> Add a sentence to the instructions for using typescript in an existing project instructing the user to copy the `paths` compiler option from the existing jsconfig file to the new tsconfig file.
> Not doing so causes absolute imports from project directories to break, and gives "Module Not Found" messages that the docs do not have a case for solving"

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 396c2cbaac56377d060b246a096b396afa8d876e

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: 396c2cbaac56377d060b246a096b396afa8d876e

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx [hunk 1]
```diff
@@ -27,7 +27,7 @@ npx create-next-app@latest
 
 ## Existing Projects
 
-Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next dev` and `next build` to automatically install the necessary dependencies and add a `tsconfig.json` file with the recommended config options.
+Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next dev` and `next build` to automatically install the necessary dependencies and add a `tsconfig.json` file with the recommended config options. Finally, copy the `paths` compiler option from your existing `jsconfig.json` file into the compiler options of the `tsconfig.json` file, and delete the `jsconfig.json` file.
```
Comment (styfle) [inline docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx:30]:
```suggestion
Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next dev` and `next build` to automatically install the necessary dependencies and add a `tsconfig.json` file with the recommended config options. 

If you already had a `jsconfig.json` file, copy the `paths` compiler option from the old `jsconfig.json` into the new `tsconfig.json` file, and delete the old `jsconfig.json` file.
```

**Review by styfle — APPROVED** [review]

**Review by styfle — CHANGES_REQUESTED** [review]
Please fix the lint error

**Comment by ShaunFerris** [comment]
> Please fix the lint error

Hi, sorry, I am relatively new to this and am not sure what is causing the lint error or how to fix it. Please advise when you get a chance.

**Comment by ShaunFerris** [comment]
> Please fix the lint error

Hi,
I have run the linting script this time so hopefully the check will not fail. It may be worth linking to the contributing documentation from the Docs Contribution Guide, as there is currently no mention of getting the dev environment setup or running the lint script there.

**Review by styfle — APPROVED** [review]
Thanks!

**Comment by ShaunFerris** [comment]
If any maintainer could offer advice on why these checks might be failing I would appreciate it. I apologize if I'm missing something obvious, Cheers!

**Comment by styfle** [comment]
GitHub is having an outage https://www.githubstatus.com/incidents/6503rcn8s34s

**Comment by styfle** [comment]
The "Validate Docs" was broken in a previous PR. 

I think the check is actually wrong because the link looks correct. cc @delbaoliveira

**Comment by styfle** [comment]
> It may be worth linking to the contributing documentation from the Docs Contribution Guide, as there is currently no mention of getting the dev environment setup or running the lint script there.

It would be great if you could submit another PR to improve that guide now that you have gone through the process and know what's missing. Thanks!

**Comment by ShaunFerris** [comment]
> > It may be worth linking to the contributing documentation from the Docs Contribution Guide, as there is currently no mention of getting the dev environment setup or running the lint script there.
> 
> Thanks for pointing that out!c It would be great if you could submit another PR to improve that guide now that you have gone through the process and know what's missing. Thanks!

Cool I was thinking that I would do that, but just to clarify while I have you here, do you think the line "To contribute, you can edit the files directly on GitHub or clone the repo and edit the files locally." should be changed to remove the option of editing directly on github, as you cant run the linter from githubs editor? (Unless I am missing something).

Cheers for the feedback, I am keen to continue contributing!

**Comment by styfle** [comment]
> as you cant run the linter from github

Maybe that's the real problem. I bet there is a way to automatically run lint in an action and commit the result.

**Comment by styfle** [comment]
I checked the PR template and I see that it does mention prettier fix here: https://github.com/vercel/next.js/blob/e048f4ef037d7eaff765adb2867d219a43a333d9/.github/pull_request_template.md#L9

**Pull request merged**